Xamarin.Forms.Core/NativeBindingConverterAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace Xamarin.Forms
+{
+	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+	public sealed class NativeBindingConverterAttribute : HandlerAttribute
```

Comment:
We don't need this at this stage, we should first make sure normal converters works